### PR TITLE
Update planning.md

### PR DESCRIPTION
**Pull request by floradanna**
- [x] New keywords
- [x] Typo:  research sofTwares
- [x] Remove the paragraph about DMP tools which overlaps  with problem page "data management plan"
- [x] "Related topics" replaced by “Problems to be addressed in this stage”
- [x] Place training after “Problems to be addressed in this stage”
- [x] Improve content based on contentathon review

### CODE REVIEW

File: pages/rdm_cycle/planning.md [hunk 1]
```diff
@@ -1,61 +1,56 @@
...
-keywords:
-contributors: [Siiri Fuchs, Korbinian Bösl, Minna Ahokas, Federico Bianchini]
+keywords: [dmp, data stewardship]
+contributors: [Siiri Fuchs, Korbinian Bösl, Minna Ahokas, Federico Bianchini, Flora D'Anna]
 ---
 
 ## What is Data Management Planning?
 
-Data management planning means to formally outline solutions regarding the handling and storing of the data generated within a project, both as the activity is running and after its conclusion. The first step is the creation of Data Management Plan (DMP), a document reporting in detail the data types that will be generated, the associated metadata, the plan for data preservation, the type of analysis that will be performed, data ownership and licensing.
+Data Management Planning consists in defining the strategy that you plan to use for managing data and documentation generated within the project. It is about thinking upfront what's the best way to avoid problems or unexpected costs related to data management, and set the conditions for your research data to achieve the highest possible impact in science, even after the end of the project.
```
Comment (smza) [inline pages/rdm_cycle/planning.md:9]: of

**Review by wittigue — APPROVED** [review]

File: pages/rdm_cycle/planning.md [hunk 2]
```diff
@@ -1,61 +1,56 @@
...
+
+Solutions regarding the handling of the data generated within a project is usually formalised in a Data Management Plan (or DMP). A DMP is a document describing several aspects of the data management process which occur before, during and after the end of a project.
```
Comment (smza) [inline pages/rdm_cycle/planning.md:11]: delete or from (or DMP)

File: pages/rdm_cycle/planning.md [hunk 3]
```diff
@@ -1,61 +1,56 @@
...
 
-Data management planning:
+## What should be considered for Data Management Planning?
 
-* helps to plan and budget necessary resources and equipment.
-* defines roles and responsibilities in data management among the project team.
-* helps to identify risks in data handling and apply solutions at early stage.
-* facilitates data sharing, reuse and preservation.
-* is often a requirement of research organisations and funders.
+* Research organisation and funders often require a DMP as part of the application for grants or, latest, when the project is funded. Therefore, consider guidelines, policies and tools for data management planning required by your funder.
```
Comment (smza) [inline pages/rdm_cycle/planning.md:34]: remove comma after grant or ","

**Review by smza — COMMENTED** [review]
Just a few typos that need to be corrected

File: pages/rdm_cycle/planning.md (hunk 1 above)
Comment (floradanna) [reply pages/rdm_cycle/planning.md:9]: done

File: pages/rdm_cycle/planning.md (hunk 2 above)
Comment (floradanna) [reply pages/rdm_cycle/planning.md:11]: done

File: pages/rdm_cycle/planning.md (hunk 3 above)
Comment (floradanna) [reply pages/rdm_cycle/planning.md:34]: done

**Comment by floradanna** [comment]
@smza Thanks a lot for the review!

**Pull request merged**